topics/variant-analysis/tutorials/exome-seq/workflows/workflow_exome_seq_from_premapped.ga

```diff
@@ -114,207 +114,186 @@
             "type": "data_input"
         },
         "4": {
-            "tool_id": "toolshed.g2.bx.psu.edu/repos/devteam/bamtools_filter/bamFilter/2.4.1",
```

Comment:
anything from with bamFiler?

Reply:
No, but the somatic variant tutorial uses BamFilter so I thought we could demonstrate another way of filtering here.